spec/models/upload_spec.rb

```diff
@@ -618,4 +618,38 @@ def enable_secure_media
       expect(Upload.secure_media_url?(url)).to eq(false)
     end
   end
+
+  describe "dominant_color" do
```

Comment:
```suggestion
  describe "#dominant_color" do
```

Reply:
I also think it'll be helpful if we assert for the unhappy paths, timeout when running the commands etc..

Reply:
Added some more specs for the unhappy paths :+1: